p5/sketch/userspace.py

```diff
@@ -283,19 +283,15 @@ def exit(*args, **kwargs):
     that necessary cleanup steps are performed before exiting the
     sketch.
 
-    :param args: positional argumets to pass to Python's builtin
+    :param args: positional arguments to pass to Python's builtin
```

Comment:
Docstring should be changed now, given we are now not calling the python's builtin function.